core/src/main/resources/reference.conf

```diff
@@ -1,5 +1,9 @@
 enableDefaultRedirect = false
+redirectDomains     = []
 enableStartupChecks = true
+terminationDeadline = 10.seconds
+preTerminationPeriod = 10.seconds
+preTerminationUnhealthy = false
```

Comment:
Couldn't find any documentation on these. Especially curious why we want `preTerminationUnhealthy` to be `false` by default.

Reply:
There is [some documentation in the example hocon files](https://github.com/snowplow/stream-collector/blob/feature/graceful-shutdown/examples/config.kinesis.extended.hocon#L259-L268).

For why it's false by default.... I have been told that it in a typical kubernetes setup we don't really need the health endpoint to return 503 during shutdown.  It's completely OK for the health endpoint to always return 200, even when shutting down.  I am adding the feature because I am told it can be helpful for debugging shutdown problems.